src/platform_impl/linux/x11/mod.rs

```diff
@@ -543,14 +541,14 @@ impl<T> EventLoopWindowTarget<T> {
         &self.xconn
     }
 
-    pub fn set_device_event_filter(&self, filter: DeviceEventFilter) {
+    pub fn listen_device_events(&self, filter: DeviceEvents) {
```

Comment:
to `filter` or not to `filter` ?
seems like names change randomly.

Reply:
See https://github.com/rust-windowing/winit/pull/2700#discussion_r1125589786, but the current change should probably remove all mentions of "filter".

Reply:
Makes sense, I'll change it to `allowed`.